docs/blog/posts/bazzite-buzz-03.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2023-08-26
+comments: true
+authors: 
+  - nicknamenamenick
+links:
+  - Bazzite Commits: https://github.com/ublue-os/bazzite/commits/main
+  - Bazzite 1.0: https://universal-blue.org/blog/2023/08/20/bazzite-10/
+---
+[comment]: <> (last commit I wrote here: f46de9b)
+
+# Bazzite Buzz No. 3
+[command]: <> (*Words, images, videos, GIFs, go here*)
```

Comment:
Are these placeholders that should be removed?

Reply:
Yeah I guess they can.  They shouldn't show up, I think, as they were just there for my own guide, and actually the commit is wrong, I just did the latest one today.  Yeah I can remove them.